easybuild/easyconfigs/c/Clang/Clang-10.0.0-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,68 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2013-2015 Dmitri Gribenko, Ward Poelmans
+# Authors:: Dmitri Gribenko <gribozavr@gmail.com>
+# Authors:: Ward Poelmans <wpoely86@gmail.com>
+# License:: GPLv2 or later, MIT, three-clause BSD.
+# $Id$
+##
+
+name = 'Clang'
+version = '10.0.0'
+
+homepage = 'https://clang.llvm.org/'
+description = """C, C++, Objective-C compiler, based on LLVM.  Does not
+ include C++ standard library -- use libstdc++ from GCC."""
+
+# Clang also depends on libstdc++ during runtime, but this dependency is
+# already specified as the toolchain.
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+# Do not set optarch to True: it will cause the build to fail
+toolchainopts = {'optarch': False}
+
+source_urls = ["https://github.com/llvm/llvm-project/releases/download/llvmorg-%(version)s"]
+sources = [
+    'llvm-%(version)s.src.tar.xz',
+    'clang-%(version)s.src.tar.xz',
+    'compiler-rt-%(version)s.src.tar.xz',
+    'polly-%(version)s.src.tar.xz',
+    'openmp-%(version)s.src.tar.xz',
+    # Also include the LLVM linker
+    'lld-%(version)s.src.tar.xz',
+    'libcxx-%(version)s.src.tar.xz',
+    'libcxxabi-%(version)s.src.tar.xz',
+]
+# patches = ['libcxx-%(version)s-ppc64le.patch']
```

Comment:
May I ask why is this patch commented out? is it no longer necessary or does it need an update?

Reply:
I could not test if it is still required on PPC arch nor if it is stil applicable, thus I leave it out. If you have a way to test it?

Reply:
@Falkor I checked that patch and it still applies to Clang v10. I made a PR to your branch to help you update this easyconfig. Please check https://github.com/ULHPC/easybuild-easyconfigs/pull/1

Reply:
Thanks @lexming -- it's merged and I added the checksum